Framework/Source/API/ConstantBuffer.cpp

```diff
@@ -352,7 +294,7 @@ namespace Falcor
             if (numMembers > 1)
             {
                 // display information for specific index of array
-                int32& refGuiArrayIndex = guiArrayIndices[currentStructName + displayName];
+                int32& refGuiArrayIndex = sGuiArrayIndices[currentStructName + displayName];
```

Comment:
When the `unordered_map` value type is `int32_t` you can just feed `sGuiArrayIndices[currentStructName + displayName]` into `addIntVar` because `operator[]` returns a reference